api-admin/src/main/java/bio/terra/pearl/api/admin/controller/dataimport/ImportController.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.pearl.api.admin.controller.dataimport;
+
+import bio.terra.pearl.api.admin.api.DataImportApi;
+import bio.terra.pearl.api.admin.service.AuthUtilService;
+import bio.terra.pearl.api.admin.service.enrollee.EnrolleeImportExtService;
+import bio.terra.pearl.core.model.admin.AdminUser;
+import jakarta.servlet.http.HttpServletRequest;
+import java.io.DataInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class ImportController implements DataImportApi {
+  private AuthUtilService authUtilService;
+  private HttpServletRequest request;
+  private EnrolleeImportExtService enrolleeImportExtService;
+
+  public ImportController(
+      AuthUtilService authUtilService,
+      HttpServletRequest request,
+      EnrolleeImportExtService enrolleeImportExtService) {
+    this.authUtilService = authUtilService;
+    this.request = request;
+    this.enrolleeImportExtService = enrolleeImportExtService;
+  }
+
+  @Override
+  public ResponseEntity<Object> delete(
+      String portalShortcode, String studyShortcode, String envName, Integer importId) {
+    return DataImportApi.super.delete(portalShortcode, studyShortcode, envName, importId);
+  }
+
+  @Override
+  public ResponseEntity<Object> get(String portalShortcode, String studyShortcode, String envName) {
+    return DataImportApi.super.get(portalShortcode, studyShortcode, envName);
+  }
+
+  @Override
+  public ResponseEntity<Object> callImport(
+      String portalShortcode,
+      String studyShortcode,
+      String envName,
+      String importFileName,
+      Object body) {
+    AdminUser operator = authUtilService.requireAdminUser(request);
+    try {
+      InputStream inputStream = new DataInputStream(new FileInputStream(importFileName));
```

Comment:
this will result in a FileNotFound exception since the file doesn't exist on the server.  Instead, I think you want to model after the SiteMediaController::upload method, and use a MultipartFile.  then you can just call `getInputStream()` on the MultipartFile